canonical/canonical.com
# Build sustainability page

## Done

- Built sustainability page as per [design](https://www.figma.com/file/KQSCmTqOjY4gfKvJPyvzj2/Canonical.com-latest-only?node-id=7390%3A21105&t=yLxIeHDfRahCjLgS-0) and [copy doc](https://docs.google.com/document/d/1wwSOXczrGS45e1DiXbt4GMsHKolniRlywxV_CNHHdCA/)
- Upgrade to latest vanilla version (needed to fix list text wrapping issue) 
 

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: https://canonical-com-809.demos.haus/careers/sustainability
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check against design and copy doc 

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-1818
